Makefile

```diff
@@ -69,7 +69,7 @@ build-mdbook:  ## Build mdbook.
 	mdbook build
 
 .PHONY: build-storybook
-build-storybook: install generate-api-types  ## Build storybook.
+build-storybook:  ## Build storybook.
```

Comment:
I don't really get this change, both dependencies are needed? And furthermore `npm run doc`, is needed, too.